### PR TITLE
Add functional tests

### PR DESCRIPTION
**Pull request by mauricioabreu**
Improving code coverage by adding more functional tests.

### CODE REVIEW

File: deck/tests/test_functional.py [hunk 1]
```diff
@@ -258,6 +258,24 @@ def test_event_create_event_proposal_with_passed_due_date(self):
...
+        response = self.client.get(
+            reverse('create_event_proposal', kwargs={'slug': event.slug}))
+        self.assertEqual(302, response.status_code)
+
+    def test_export_event_votes_to_csv(self):
+        event = Event.objects.create(**self.event_data)
+        response = self.client.get(
+            reverse('export_event', kwargs={'slug': event.slug}))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response['Content-Type'], 'text/csv')
```
Comment (luanfonceca) [inline deck/tests/test_functional.py]: We need to assert the content too in here.

Comment (mauricioabreu) [inline deck/tests/test_functional.py]: You are right! Thank you!

Comment (mauricioabreu) [inline deck/tests/test_functional.py]: Ok, I tried without success. What should be the attribute to assert the CSV content?
I think we should move that query (inside the export to CSV view) and test it as an unit test.

Comment (luanfonceca) [inline deck/tests/test_functional.py]: It is even better, because we move the logic to the model.

Comment (luanfonceca) [inline deck/tests/test_functional.py]: @mauricioabreu i was looking at the library that we use and i saw a test folder: https://github.com/azavea/django-queryset-csv/tree/master/test_app/djqscsv_tests/tests

Maybe we can found something useful in there.

o/

Comment (mauricioabreu) [inline deck/tests/test_functional.py]: I saw that code before saying that "I tried without success" and my conclusion is: too many magic tricks in there. :PPP
Any ideas? Or let's move on? You call.

Comment (luanfonceca) [inline deck/tests/test_functional.py]: Nope, its ok for me, lets merge this. 

o/

**Pull request merged**